### PR TITLE
feat(region): Add additional region mismatch info

### PR DESCRIPTION
**Pull request by magsharon**
Add additional region mismatch info

Ticket Link: https://www.notion.so/immersve/Document-KYC_STATEMENT_REGION_MISMATCH-error-code-in-public-docs-9a77f54fe9c04ff4874324dc07018d42

### CODE REVIEW

**Review by rb455 — COMMENTED** [review]
It looks like the task is about a different error code: `KYC_STATEMENT_REGION_MISMATCH`, not `KYC_PROFILE_REGION_MISMATCH`

Also, we normally use OpenAPI reference for documenting error codes, not the guide.

image

File: imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml [hunk 1]
```diff
@@ -147,6 +147,9 @@ post:
 
         **UNSUPPORTED_KYC_REGION**
         The KYC region is not yet supported.
+
+        **KYC_STATEMENT_REGION_MISMATCH**
+        The latest KYC statement is either not submitted or requires manual review, and the region is different from the requested region.
```
Comment (rb455) [inline imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: there are 2 issues with this wording:
- there is a logical error: not submitted or requires manual review -> not submitted or **does not** require manual review
- even with the above correction it is looking to complex to understand, suggesting a simpler wording:
"The latest unchecked KYC statement is for a region that is different from the requested region."

**Review by rb455 — APPROVED** [review]

File: imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml [hunk 2]
```diff
@@ -147,6 +147,9 @@ post:
 
         **UNSUPPORTED_KYC_REGION**
         The KYC region is not yet supported.
+
+        **KYC_STATEMENT_REGION_MISMATCH**
+        The latest unchecked KYC statement is for a region that is different from the requested region.
```
Comment (catluin) [inline imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: just curious, what does "unchecked" mean in this context?

Comment (magsharon) [reply imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: Great question! By 'unchecked,' it means that the latest KYC statement hasn't been submitted yet or doesn't require manual review.  If that's the case and the region is different from the requested one, that's when we return the KYC_STATEMENT_REGION_MISMATCH error. Thanks for pointing that out! You're right—we should rephrase that part to make it clearer. Any thoughts @rb455 ?

Comment (catluin) [reply imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: I guess it's a bit confusing cos I'm not very familiar with the domain and KYC statement lifecycle :) Do we create a blank KYC statement first and the attach supporting documents to it, and that's when it becomes "submitted"?
We don't need to reiterate it all here but adding a back-link to a corresponding KYC guide might be helpful

Comment (rb455) [reply imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: I don't see any suggestion to rephrase it in Kira's comment, she is just curious, right?

'unchecked' means that the latest KYC statement from the partner hasn't been submitted to FrankieOne for a KYC verification yet

Comment (rb455) [reply imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: 'unchecked' means that the latest KYC statement from the partner hasn't gone through a KYC verification yet. Submitting a statement is separate from initiating the check.

Comment (rb455) [reply imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: adding a backlink? i don't mind, won't hurt

Comment (catluin) [reply imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml:152]: > 'unchecked' means that the latest KYC statement from the partner hasn't gone through a KYC verification yet. Submitting a statement is separate from initiating the check. The check is initiated by calling spending prerequisites

@rb455 thanks, that makes much more sense now.

**Review by catluin — APPROVED** [review]

**Pull request merged**